packages/jw-player/src/components/jw-player/index.css.js

```diff
@@ -0,0 +1,89 @@
+import { css } from 'lit';
+
+export const style = css`
+  :host([captions]) #container {
+    grid-template-rows: 1fr max-content 1fr;
+  }
+
+  [active] {
+    background-color: var(--lumo-shade-10pct);
+  }
+
+  [match]:not([active]) {
+    background-color: var(--lumo-primary-color);
+  }
+
+  .captions span {
+    cursor: pointer;
```

Comment:
```suggestion
    cursor: default;
```

Seeing hand / pointer on all page, makes it odd experience for me at least.

Reply:
If an element performs an action, such as a button or a link, I would expect the cursor to turn into a pointer. This let's the user know that clicking the element will do something. I can get rid of the pointer, though it feels unnatural to me.